Explicitamente cria o diretório /home/bank/Downloads

filipesaraiva opened the pull request:
Atualmente o Firefox reclama de falta de permissão para salvar em Downloads. Criando
explicitamente esse diretório antes do comando de atribuição de permissão corrige o erro

filipesaraiva committed:
Explicitamente cria o diretório /home/bank/Downloads

Atualmente o Firefox reclama de falta de permissão para salvar em Downloads. Criando
explicitamente esse diretório antes do comando de atribuição de permissão corrige o erro

diraol commented on warsaw-browser/Dockerfile:
```diff
@@ -38,6 +38,7 @@ RUN apt update \
   && groupadd -g 1000 -r bank \
   && useradd -u 1000 -r -g bank -G audio,video bank \
   && mkdir -p /home/bank \
+  $$ mkdir -p /home/bank/Downloads \
```
1) Tem um typo na linha... deveria ser `&&` ao invés de `$$`.

2) Se você fizer apenas: `mkdir -p /home/bank/Downloads` já é o suficiente, o comando anterior é completamente desnecessário.

filipesaraiva replied:
Obrigado pela observação @diraol. Corrigido!

filipesaraiva committed:
Substitui $$ por &&

Corrige o typo e utiliza apenas um comando para criar a pasta /home/bank e /home/bank/Downloads

Obrigado @diraol.
